package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viamrobotics/prime",
-  "version": "0.0.101",
+  "version": "0.0.102",
```

Comment:
Not an issue for this but I think we need to start considering what is a major/minor/patch and updating this accordingly. We have definitely added new APIs in the way of new elements/components and never did minor bump.

Reply:
Yep! agreed that we should start proper versioning at this point.